.github/workflows/lint_python.yml

```
@@ -9,7 +9,7 @@ jobs:
       - run: pip install bandit black codespell flake8 isort mypy pytest pyupgrade safety
       - run: bandit --recursive --skip B101 .  # B101 is assert statements
       - run: black --check . || true
-      - run: codespell --ignore-words-list="ans,nnumber" --quiet-level=2  # --skip=""
+      - run: codespell --ignore-words-list="ans,nnumber,nin" --quiet-level=2  # --skip=""
```

```suggestion
      - run: codespell --ignore-words-list="ans,nin,nnumber" --quiet-level=2  # --skip=""
```
Keeping the values sorted make it easy to find value and makes it difficult to add duplicate values.  This is easy when there are few values but gets more useful as lists get longer.